src/main/scala/nl.knaw.dans.easy.validatebag/rules/metadata/package.scala

```diff
@@ -389,6 +390,32 @@ package object metadata extends DebugEnhancedLogging {
     }
   }
 
+  def filesXmlFilesHaveOnlyAllowedAccessRights(t: TargetBag): Try[Unit] = {
+    trace(())
+    for {
+      xml <- t.tryFilesXml
+      files <- getFileElements(xml)
+      _ <- validateFileAccessRights(files)
+    } yield ()
+  }
+
+  private def getFileElements(xml: Elem): Try[NodeSeq] = Try {
+    xml \ "file"
+  }
+
+  private def validateFileAccessRights(files: NodeSeq): Try[Seq[Unit]] = {
+    files.map(validateAccessRights).collectResults.recoverWith {
+      case ce: CompositeException => Try(fail(ce.getMessage))
+    }
+  }
+
+  private def validateAccessRights(file: Node): Try[Unit] = Try {
+    val accessRights = file \ "accessRights"
+    accessRights.foreach(rights =>
```

Comment:
This assumes (1) there can be multiple `<accessRights/>` elements within one `<file/>` element and (2) that only one of them can have an invalid value: once you found an invalid value, you skip the others.

First see if (1) is even possible (I honestly don't know) and if that's the case, make it such that all invalid values are caught.